Assets/PatchKit Patcher/Scripts/AppUpdater/AppRepairer.cs

```diff
@@ -162,17 +159,16 @@ int installedVersionId
 
         private IEnumerable<FileIntegrity> FilesNeedFixing(VersionIntegrity results)
         {
-            var missingFiles = results.Files.Where(f => f.Status == FileIntegrityStatus.MissingData);
-            var invalidSizeFiles = results.Files.Where(f => f.Status == FileIntegrityStatus.InvalidSize);
+            var invalidFiles = results.Files.Where(f => f.Status != FileIntegrityStatus.Ok);
 
-            return missingFiles.Concat(invalidSizeFiles);
+            return invalidFiles;
         }
 
         private long CalculateRepairCost(AppContentSummary contentSummary, IEnumerable<FileIntegrity> filesToRepair)
         {
             return filesToRepair
                 .Select(f => contentSummary.Files.FirstOrDefault(e => e.Path == f.FileName))
-                .Sum(f => f.Size);
+                .Sum(f => Math.Max(PACK1_BLOCK_SIZE, f.Size));
```

Comment:
I've just checked that the chunk size is always defined inside the content or diff summary. You can access it here using `contentSummary.Chunks.Size`. This means that you can remove that constant.

Reply:
Done